api/src/pcapi/core/educational/utils.py

```diff
@@ -0,0 +1,10 @@
+from datetime import datetime
+
+from pcapi.core.educational import constants
+
+
+def compute_cancellation_limit_date_for_educational_booking(
+    event_beginning: datetime, booking_creation_date: datetime
+) -> datetime:
+    fifteen_days_before_event = event_beginning - constants.CONFIRM_EDUCATIONAL_BOOKING_BEFORE_EVENT_DELAY
```

Comment:
Un peu bizarre d'avoir le nombre de jour dans le nom de la variable alors que l'as mis en constante 🤔

Reply:
Il faudrait pas choper la timezone du lieu aussi pour avoir une date UTC correcte sur la date limite d'annulation ?

Reply:
Ah yes complètement, c'est vrai que je voulais faire ressortir la règle métier, pour que quand on lit la fonction on comprenne tout de suite ce qu'implem le code => 15 jours, plutôt que de retrouver la constante.
Mais je suis d'accord avec toi :)
Comment on pourrait faire pour ne pas avoir de magic value, mais garder une clarté dans le code (qu'on voit tout de suite de quoi on parle quand on lit la fonction)
Tu aurais une idée?

Reply:
Parceque je trouve la fonction actuelle sur individuelle pas clair du tout, et je ne voudrais pas réitérer ^^
Quand je lis ça : `return max(fifteen_days_before_event, booking_creation_date)` c'est hyper clair je trouve, on peut confirmer tout de suite ce que représente la `cancellationLimitDate` sur un booking eac